### ISSUE TITLE
Sensor System Submodules Auto Start Apply not Storing Setting

### ISSUE THREAD

**Issue by nickgaray** [opensensorhub/osh-core#237]
When configuring a Sensor System, in particular submodules, renaming the submodule or checking the auto start setting does not get applied when "Apply Changes" button is pressed.
This may affect any config setting for the submodules that are part of the Sensor System.

**Comment by alexrobin** [comment]
Yep, sorry this is so unusable in the UI. There are a bunch of things that don't work right with subsystems when created in the admin.
